File: Java/538.java
import java.util.Scanner;

/**
 *
 * 538 - Toesto Era Campo - Iván - https://github.com/wildfireOfMine
 */
public class Main {

       public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int abuelo = sc.nextInt();
        int construccion = sc.nextInt();
        while (abuelo != 0 || construccion != 0) {
            if (abuelo >= construccion) {
                System.out.println("CUERDO");
            } else {
                System.out.println("SENIL");
            }
            abuelo = sc.nextInt();
            construccion = sc.nextInt();
        }
    }
}

File: C++/538.cpp
#include <iostream>

using namespace std;

// 538 - Toesto era Campo - Iván - https://github.com/wildfireOfMine

int main()
{

    int anoYoNaci = 0;
    int anoIglesia = 0;
    cin >> anoYoNaci;
    cin >> anoIglesia;
    while (anoYoNaci != 0 || anoIglesia != 0) {
        if (anoYoNaci>=anoIglesia) {
            cout << "CUERDO" << endl;
        } else {
            cout << "SENIL" << endl;
        }
        cin >> anoYoNaci;
        cin >> anoIglesia;
    }
    return 0;

}
